requirements.txt

```diff
@@ -1,5 +1,5 @@
-dask>=2022.03.0
-distributed>=2022.03.0
+dask==2022.05.1
+distributed==2022.05.1
```

Comment:
These will ensure `dask` & `distributed` are pinned correctly in the Conda packages. In PR ( https://github.com/rapidsai/dask-cuda/pull/893 ), we ensure these are pulled into the Conda packages